src/components/BaseMap/BaseMap.tsx

```
@@ -79,27 +75,24 @@ const handleError = (
   }))
 }
 
-function WatershedLayers({ layer, index }) {
+function WatershedLayers({ layer, index, selectedRegion, selectedYear }) {
```

ESLint

[error] 'selectedRegion' is defined but never used.

(@typescript-eslint/no-unused-vars)